app_flair.py

```diff
@@ -51,13 +61,16 @@
 
 global tagger_pos
```

Comment:
For the globals, you could use an object on which you set and get the networks. That looks much better imho than global variables.